.github/workflows/ec2-integration-test.yml

```diff
@@ -5,9 +5,8 @@ name: Reusable EC2 Integration Test
 
 env:
   PRIVATE_KEY: ${{ secrets.AWS_PRIVATE_KEY  }}
-  TERRAFORM_AWS_ASSUME_ROLE: ${{ secrets.TERRAFORM_AWS_ASSUME_ROLE }}
-  S3_INTEGRATION_BUCKET: ${{ secrets.S3_INTEGRATION_BUCKET }}
   KEY_NAME: ${{ secrets.KEY_NAME }}
+  TERRAFORM_AWS_ASSUME_ROLE_DURATION: 14400 # 4 hours
```

Comment:
Required when timeout of step for apply hits 1 hour, so the next step destroy will function.